code/modules/food_and_drinks/food/snacks_soup.dm

```diff
@@ -225,3 +225,11 @@
 	bonus_reagents = list("nutriment" = 4, "vitamin" = 6)
 	tastes = list("beet" = 1)
 	foodtype = VEGETABLES
+
+/obj/item/reagent_containers/food/snacks/soup/bisque
+	name = "bisque"
+	desc = "A classic entr�e from France."
```

Comment:
```suggestion
	desc = "A classic entrée from Space-France."
```

Reply:
byond has limited support for unicode right now, don't use fancy letters please

Reply:
You could use &eacute; (maybe (probably not))

Reply:
just avoid them all together until proper unicode support lands

Reply:
De-fancy this